
Error:
Traceback (most recent call last):
  File "C:\AVRL\online-judge-v\online_judge\codes\2341f2f5-dea9-4fd9-bf1f-bfc8f6b284bd_code.py", line 13, in <module>
    num = int(input())
EOFError: EOF when reading a line
